geoh5py/shared/concatenation.py

```diff
@@ -137,13 +136,8 @@ def concat_attr_str(self) -> str:
         """String identifier for the concatenated attributes."""
         if self._concat_attr_str is None:
             self._concat_attr_str = "Attributes"
-            if (
-                self.workspace.ga_version is not None
-                and "4." in self.workspace.ga_version
-            ):
-                version = re.findall(r"\d+\.\d+", self.workspace.ga_version)
-                if version and float(version[0]) >= 4.2:
-                    self._concat_attr_str = "Attributes Jsons"
+            if self.workspace.version is not None and self.workspace.version > 2.0:
```

Comment:
**question:** @jincandescent, are there case where the version of Analyst matters to interpret the content of geoh5 ?

Reply:
Confirmed with Julien